src/OSPSuite.InstallationValidator.Core/Presentation/MainPresenter.cs

```diff
@@ -61,27 +68,42 @@ public void Abort()
          _cancellationTokenSource = new CancellationTokenSource();
          try
          {
-            View.ValidationIsRunning(true);
-            logText(Captions.StartingBatchCalculation);
-            await _batchStarterTask.StartBatch(_outputFolderDTO.FolderPath, _cancellationTokenSource.Token);
+            updateValidationRunningState(running: true);
+            logLine(Captions.StartingBatchCalculation);
+            var validationResult = new InstallationValidationResult {RunSummary = await _batchStarterTask.StartBatch(_outputFolderDTO.FolderPath, _cancellationTokenSource.Token)};
+
+            logLine(Captions.StartingComparison);
+            validationResult.ComparisonResult = await _batchComparisonTask.StartComparison(_outputFolderDTO.FolderPath, _cancellationTokenSource.Token);
 
-            logText(Captions.StartingComparison);
-            await _batchComparisonTask.StartComparison(_outputFolderDTO.FolderPath, _cancellationTokenSource.Token);
+            logLine(Captions.StartReport);
+            await _validationReportingTask.StartReport(validationResult, _outputFolderDTO.FolderPath);
          }
          catch (OperationCanceledException)
          {
-            logText(Captions.TheValidationWasCanceled);
+            logLine(Captions.TheValidationWasCanceled);
          }
          catch (Exception e)
          {
             logException(e);
          }
          finally
          {
-            View.ValidationIsRunning(false);
+            updateValidationRunningState(running:false);
+            logLine(Captions.ValidationCompleted);
          }
       }
 
+      private void logLine(string textToLog)
+      {
+         logText($"{textToLog}{Environment.NewLine}");
```

Comment:
Does that mean that with logtext, we could append text to the same line? That's a neat feature to add live progress with "."